src/Nox.Types/Types/Color/ColorConverter.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Nox.Types;
+
+public static class ColorConverter
```

Comment:
Can we rename this one to Helper or something like that, because we are using Converters for the EnityFramework?

Reply:
Done